NeoGeoMiniRomAdder/Form1.Designer.cs

```diff
@@ -69,47 +69,52 @@ partial class Form1 {
             this.flowLayoutPanel1.Dock = System.Windows.Forms.DockStyle.Fill;
```

Comment:
adding 
`this.ngmhFolderbrowserDlg.UseDescriptionForTitle = true;`
to the ngmhFolderbrowserDlg section will ensure the caption displays correctly, else it looks dodgy in the converted .NET core version.